client/my-sites/stats/stats-date-picker/index.jsx

```diff
@@ -75,8 +103,26 @@ class StatsDatePicker extends Component {
 		return formattedDate;
 	}
 
+	renderQueryDate() {
+		const { queryDate, moment, translate } = this.props;
+		if ( ! queryDate ) {
+			return null;
+		}
+
+		const today = moment();
+		const date = moment( queryDate );
+		const isToday = today.isSame( date, 'day' );
+		return translate( 'Last update: %(time)s', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **79** times:
`translate( 'Last updated: %s' )` *ES Score: 7.32*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).